### PR TITLE
EVK-68 Documentation for Zendar ROS Driver

### PR DESCRIPTION
**Pull request by MarcBrau**
Added README.md on main page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+#### Install Other External Dependencies
+Run the following command to install all other required dependencies: 
+```
+sudo apt install pcl protobuf-compiler libpcl-dev libgoogle-glog-dev ros-noetic-pcl-ros
+```
+
+#### Install Zendar Host Package
+Install the host package supplied by Zendar with the following command:
+```
+sudo -i package_name.deb
```
Comment (jimmy-zendar) [inline README.md:29]: sudo apt install zendar_host_latest.deb?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+### Setup zendar_ros_driver Repository
+Clone the zendar_ros_driver repository:
+```
+git@github.com:ZendarInc/zendar_ros_driver.git
+```
+Define an environment variable in your shell environment by adding the following line to your .bashrc script (typically in ~/.bashrc):
+```
```
Comment (jimmy-zendar) [inline README.md:38]: I dont think you need all these exports.

**Review by jimmy-zendar — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (MarcBrau) [reply README.md]: Addressed in 09bb536.

File: README.md (hunk 2 above)
Comment (MarcBrau) [reply README.md]: Addressed in 09bb536.

**Pull request merged**